src/main/kotlin/com/th/plu/domain/member/MemberRole.kt

```diff
@@ -0,0 +1,9 @@
+package com.th.plu.domain.member
+
+import lombok.Getter
+
+@Getter
+enum class MemberRole(private val value: String) {
```

Comment:
```suggestion
enum class MemberRole(val value: String) {
```

lombok 제거, private 제거하면 외부에서 value get 가능합니다